Gemfile

```diff
@@ -45,7 +45,7 @@ group :development, :test do
   gem 'webmock'
 end
 
-gem 'flipper', '~> 0.25.0'
```

Comment:
Did you find out the reason this was locked? I think when I tried to unlock it in sequencescape it came up with data migrations because the way they stored the flags changed after version 1.

Reply:
No idea. I will try adding a flag to make sure it still works.